_posts/2023-09-22-licensing-update.md

```diff
@@ -0,0 +1,327 @@
+MDAnalysis library under the LGPLv2.1+ license from this day going forward.
+
+Typed Signature
+```
+
+**Please note that it is important to ensure that the responding party is the copyright owner.
+In some cases, your University or Employer, at the time of contribution, may have been the
+copyright owner instead of yourself. If you think this might be the case, please let us know
+as soon as possible with any relevant information as to whom the copyright owner might be.**
+
+It is important that we hear back from as many contributors as possible. If you have contributed to
+MDAnalysis in the past but have since changed your git-linked contact details, we would kindly ask if
+you could email *licensing@mdanalysis.org* to let us know how best to contact you.
+
+### Removing Apache 2.0 dependencies
+
+As we move towards a v3.0 release of MDAnalysis, we will be looking to replace existing [Apache 2.0][]
+core dependencies. Of particular impact, this may mean that access to reading [MMTF][] files may no longer
+be available by default through the MDAnalysis library, but will instead be offered through an optionally
+installable [MDAKit](https://mdakits.mdanalysis.org/about.html).
+
+## Getting in touch
+
+We understand that license changes are a complex subject that can have a large impact on how users and
+developers may interact with the MDAnalysis library. Should you have any concerns, questions, or comments,
+we urge you to get in touch with the core developer team. You can do so over our
+[public forums]({% link index.md %}#participating) or privately by emailing *licensing@mdanalysis.org*.
+
+## Definitions
+
+### package license
+
+The license under which the MDAnalysis library is packaged and released. This may be different from
+but still compatible with the [source code license](#source-code-license).
+
+### source code license
+
+The license under which the MDAnalysis source code is released. This only accounts for the code itself
+and not any impact in licensing incurred from linking (i.e. `python importing`) to other libraries.
+
+### license compatibility
+
+Whether or not the terms of two licenses allows them to be compatible with each other.
+
+### copyleft
+
+Whether the terms of a given license enforces that other codes (even under other licenses) must be bound
+by the same conditions. See the [FSF definition](https://www.gnu.org/licenses/copyleft.en.html) for more information.
+
+## Acknowledgements
+
+We are very grateful for the administrative and legal support from our fiscal sponsor, [NumFOCUS][].
+
+This relicensing effort has been made possible in part by the CZI grant DAF2021-237663,
+grant DOI https://doi.org/10.37921/426590wiobus from the Chan Zuckerberg Initiative DAF,
+an advised fund of Silicon Valley Community Foundation (funder DOI 10.13039/100014989).
+
+– Irfan Alibay on behalf of the MDAnalysis [Core Developers][]
```

Comment:
I'm maybe being a bit much here, so I'm willing to hear what folks think / not do this. At the end of the day this has been much of a solo project that has cost me far far too much time (just this blog alone was order 15h combined with checking up legal stuff). Unlike a repo I can't really put my name on anything else but this.

Reply:
I am cool with you signing it on the behalf of the core devs.

Reply:
I'd been thinking to suggest something you do this, so I'm definitely in support of having your name there!